reactor-core/src/main/java/reactor/core/publisher/EmitterProcessor.java

```diff
@@ -54,8 +54,9 @@
  * @author Stephane Maldini
  * @deprecated To be removed in 3.5. Prefer clear cut usage of {@link Sinks} through
  * variations of {@link Sinks.MulticastSpec#onBackpressureBuffer() Sinks.many().multicast().onBackpressureBuffer()}.
- * This processor is blocking on {@link EmitterProcessor#emitNext(T)}. This behaviour can be implemented
- * with the {@link Sinks} API by calling {@link Sinks#many()#tryEmitNext(T)} and retrying, e.g.:
+ * This processor is blocking on {@link EmitterProcessor#tryEmitNext(Object)}.
```

Comment:
that javadoc is actually misleading. should be `This processor was blocking` and `#onNext(Object)`

Reply:
```suggestion
 * This processor was blocking in {@link EmitterProcessor#onNext(Object)}.
```